nixos/modules/services/networking/bee-clef.nix

```diff
@@ -0,0 +1,76 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.bee-clef;
+in {
+  meta = {
+    maintainers = with maintainers; [ attila-lendvai ];
+  };
+
+  ### interface
+
+  options = {
+    services.bee-clef = {
+      enable = mkEnableOption "clef external signer instance for Ethereum Swarm Bee";
+
+      data-dir = mkOption {
+        type = types.nullOr types.str;
+        default = "/var/lib/bee-clef";
+        description = "Data dir for bee-clef.";
```

Comment:
Descriptions should not end in `.`.

Reply:
Are you sure about that @sternenseemann? The [official documentation](https://nixos.org/manual/nixos/stable/#sec-option-declarations) uses a period at the end of a description. I'm pretty sure it should end in a period :thinking:

Reply:
Hmmmmm, [we require it for `meta.description` at least](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes), but the style in `nixos/` seems to be inconsistent. Maybe we should clarify/decide that for the future.

Reply:
You are correct that for `meta.description` there should be no period. For NixOS options, though, the documentation would be very strange if we didn't use a period. There can be paragraphs in a `description`.

Reply:
ok, i have re-added the dots, except in the meta.